exporter/stackdriverexporter/stackdriver.go

```diff
@@ -111,6 +141,21 @@ func StackdriverTraceExportersFromViper(v *viper.Viper) (tps []consumer.TraceCon
 	return
 }
 
+// ExportSpans is the method that translates OpenCensus-Proto Traces into AWS X-Ray spans.
+// It uniquely maintains
+func (sde *stackdriverExporter) pushTraceData(ctx context.Context, td data.TraceData) (int, error) {
+	agent := agentForLibraryInfo(td.Node.GetLibraryInfo())
```

Comment:
in principle LibraryInfo can be nil. While Node shouldn't be nil there was some cases in the past of that so for safety it seems reasonable to consider this possibility.

Reply:
Thanks for pointing this out. I added checks for `nil` here and similarly below. I also added more test cases to cover the various `nil` values that are possible.